### ISSUE TITLE
Dropped tx while transfer successully

### ISSUE THREAD

**Issue by AsceticBear** [darwinia-network/darwinia-common#563]
The  Metamask shows that the transaction was dropped, but the transfer was executed successfully

![image](https://user-images.githubusercontent.com/11801722/112271537-03c34f00-8cb6-11eb-8e8a-af190c601f23.png)

**Comment by AsceticBear** [comment]
```sh
curl --location --request POST 'https://pangolin-rpc.darwinia.network/' \
--header 'Content-Type: application/json' \
--data-raw '{
	"jsonrpc":"2.0",
	"method":"eth_getTransactionByHash",
	"params":[
		"0x914d7f5baddc04e5343024146b3a9d28e435070679892e8bf5ab93cfc584f2a1"
	],
	"id":1
}'
```
The output: 

```sh
{“jsonrpc”:“2.0",“result”:null,“id”:1}
```

**Comment by AsceticBear** [comment]
From Metamask:
```js
 /**
    checks to see if if the tx's nonce has been used by another transaction
    @param txMeta {Object} - txMeta object
    @emits tx:dropped
    @returns {boolean}
  */
  async _checkIftxWasDropped (txMeta) {
    const { txParams: { nonce, from }, hash } = txMeta
    const nextNonce = await this.query.getTransactionCount(from)
    const { blockNumber } = await this.query.getTransactionByHash(hash) || {}
    if (!blockNumber && parseInt(nextNonce) > parseInt(nonce)) {
        return true
    }
    return false
  }
```

**Comment by AsceticBear** [comment]
The pangolin network version 210 has this issue. This issue can be fixed by #559 and wait for version 211 afterward. Close it now.

**Comment by AsceticBear** [comment]
![image](https://user-images.githubusercontent.com/11801722/112425530-33835d00-8d71-11eb-9f8e-4f1803f397b1.png)

Testing based on pangolin 211, I send five transactions in a row without interruption, and the fourth metamask shows drop, but the transfer is successful.
